test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -2044,6 +2044,93 @@ BOOST_AUTO_TEST_CASE(external_visibility)
 	CHECK_ERROR(sourceCode, DeclarationError, "Undeclared identifier.");
 }
 
+BOOST_AUTO_TEST_CASE(similar_name_suggestions_expected)
+{
+	char const* sourceCode = R"(
+		contract c {
+			function func() {}
+			function g() public { fun(); }
+		}
+	)";
+	CHECK_ERROR(sourceCode, DeclarationError, "Undeclared identifier. Did you mean func?");
```

Comment:
Can you enclose the suggestions in quotes - that is how other messages are? eg `"func"`, `"func1", "func2" or "func3"`